### PR TITLE
Update testing documentation with flake8

### PR DESCRIPTION
**Pull request by mkemel**
Update testing documentation with flake8 instead of autopep

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -148,12 +148,11 @@ execution result directory, for example:
...
 ### Code Style
 
-[autopep8](https://pypi.org/project/autopep8/) is used to enforce a unified code style. The applied rules are defined in
-the [.pep8](./.pep8) file. All source files can be formatted via:
+[flake8](https://pypi.org/project/flake8/) is used to enforce a unified code style. All source files formatting can be checked via:
 
 ```shell
 # navigate into bluechi/tests directory
-autopep8 ./
+find . -name '*.py' | xargs flake8
```
Comment (mwperina) [inline tests/README.md]: Won't you get the same results by just running `flake8` from the `tests` directory?

Comment (mkemel) [reply tests/README.md]: Turns out that yeah, the same :)

**Review by mwperina — APPROVED** [review]
LGTM

**Review by engelmi — APPROVED** [review]
LGTM

Failed integration test was probably a GH CI hickup... restarted it and ready to merge as soon as it completes.

**Pull request merged**